src/UpdatesPublisher.js

```diff
@@ -0,0 +1,78 @@
+const {createProducer} = require('windbreaker-service-util/queue')
+const ChangesStream = require('changes-stream')
+const Promise = require('bluebird')
+
+let producerOptions = null
+let amqUrl = null
+let logger = null
+let registryUrl = null
+
+class UpdatesPublisher {
+  configure (_producerOptions, _amqUrl, _registryUrl, _logger) {
```

Comment:
Quick tip: As a general rule of thumb, if your function takes in more than 2-3 arguments, you should consider making your function accept a single `options` object that contains the arguments you need. This ends up making code easier to read.

ex.
`definition`
```js
configure (options) {
  const { producerOptions, amqUrl, registryUrl, logger } = options
  // do work
}

// or
configure ({ producerOptions, amqUrl, registryUrl, logger }) {
  // do work 
}
```

`usage`
```js
publisher.configure({
  producerOptions: {},
  amqUrl: 'urlA', 
  registryUrl: 'urlB',
  logger: console
})

// vs
publisher.configure({}, 'urlA', 'urlB', console) 
// ^^^ easier to make mistakes and misplace args here
// also, I need to look at the definition to tell exactly what each arg is used for,
// it's much more obvious what is going on in the `options` arg version
```